ansible/group_vars/nimbus.mainnet.yml

```
@@ -31,7 +31,9 @@ beacon_node_subscribe_all: true
 # HTTP RPC support is unstable
 beacon_node_web3_urls: '{{ beacon_node_web3_urls_all | reject("match", "^http://") }}'
 # Light client data
-beacon_node_light_client_data_enabled: '{{ node.branch in ["unstable", "libp2p"] }}'
+beacon_node_light_client_data_enabled: true
+beacon_node_light_client_data_serve: true
+beacon_node_light_client_data_import: 'full'
```

```suggestion
beacon_node_light_client_data_import_mode: 'full'
```

Keep in mind, the actual launch arguments have been renamed on `unstable`, so may want to wait until the renames are also on `stable`, should be in 22.05 release

---

https://github.com/status-im/nimbus-eth2/pull/3740

---

Note: `beacon_node_light_client_data_serve` is now `true` by default: https://github.com/status-im/nimbus-eth2/pull/3921

---

Note: `beacon_node_light_client_data_serve` is now `true` by default: https://github.com/status-im/nimbus-eth2/pull/3550

`beacon_node_light_client_data_enabled` is still needed (this toggles whether the LC data specific CLI options are passed to the beacon node; without this, the overrides don't have any effect)